UnityProject/Assets/Scripts/UI/Systems/IngameMenu/GUI_IngameMenu.cs

```diff
@@ -141,6 +154,16 @@ public void InitiateRestartVote()
 		CloseMenuPanel();
 	}
 
+	public void GoToWiki()
+	{
+		Application.OpenURL("https://unitystation.github.io/unitystation-wiki/");
```

Comment:
it'd be best if this wasn't hardcoded. Can you get a link from ``GameConfig.json`` instead?